src/components/property-box/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { Color } from '../colors';
+import { Copy } from '../copy';
+import { Headline } from '../headline';
+import * as React from 'react';
+import { getSpace, SpaceSize } from '../space';
+import styled from 'styled-components';
+
+export interface PropertyBoxProps {
+	headline?: string;
```

Comment:
Let's make `headline` and `copy` required

Reply:
Fixed that!